### DISCUSSION TITLE
Error: Cannot find module 'typescript'

### DISCUSSION THREAD

**Discussion by nhattan097** [vercel/vercel#8055] [Help]
I push commit and get this error below, when I re-deploy by hand and it's work normally.
![image](https://user-images.githubusercontent.com/34256997/176636353-a5bf48f6-bb27-4f04-aee9-eaac4009a1f7.png)

**Comment by amyegan** [comment]
When you re-deploy by hand, is that from the Vercel Dashboard or a CLI deployment from your computer? Without looking at your project I can't say for sure what's happening, but I would be happy to look into this issue more if you can share your repo or a minimal reproduction of the problem. The folks in [nx discussions](https://github.com/nrwl/nx/discussions) might have some ideas if you want to ask over there as well.

**Reply by mkontsek** [reply]
this started happening to us when we added Ignored Build Step `npx nx-ignore my-app`

**Reply by amyegan** [reply]
Can you tell me more about what steps you are taking to "re-deploy by hand" and share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example)? I'm happy to help you debug, but I need to be able to see what is causing the error.

**Reply by mkontsek** [reply]
at the moment Vercel is cloning the feature branch into a new local git repo named `master`, so there is no way of knowing what changes occured to enable nx checking or any kind of git analysis. This issue needs to be fixed on Vercel before going forward (or it's no longer a feature provided by Vercel).

**Reply by amyegan** [reply]
I'm sorry you're still having trouble @mkontsek. In the error printout, it mentions using `--skip-nx-cache`, which has solved build problems with nx repos for other people recently (https://github.com/vercel/community/discussions/816). I would recommend giving that a try.

If you can describe the steps and Project settings that are leading to your successful and unsuccessful builds, then I can be more helpful in identifying exactly what is the going wrong. The Build Engineers will need to know more replication details if this is a bug.
